pkg/cluster/config/types.go

```diff
@@ -99,7 +101,20 @@ type Networking struct {
 	// PodSubnet is the CIDR used for pod IPs
 	// kind will select a default if unspecified
 	PodSubnet string
+	// ServiceSubnet is the CIDR used for services VIPs
+	// kind will select a default if unspecified for IPv6
```

Comment:
```suggestion
	// kind will select a default if unspecified
```

we're always selecting this. not a big deal though